Gemfile.lock

```diff
@@ -616,69 +619,68 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.5.0)
     snaky_hash (2.0.1)
       hashie
       version_gem (~> 1.1, >= 1.1.1)
     socksify (1.7.1)
-    soft_deletion (1.6.0)
-      activerecord (>= 4.2.0, < 6.2.0)
+    soft_deletion (1.9.0)
```

Comment:
### [soft_deletion](https://github.com/grosser/soft_deletion)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.6.0 → 1.9.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/soft_deletion/1.6.0/1.9.0)_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/grosser/soft_deletion/compare/v1.6.0...v1.9.0).

These are the first 10 commits:
- (fc8c9c8) [drop EOL dependencies](https://github.com/grosser/soft_deletion/commit/fc8c9c80d57aeb90f2b471383377d297f43e8d52)
- (2fb3569) [more](https://github.com/grosser/soft_deletion/commit/2fb356997a23fc0f7544c81ba8a92fc969eb86a6)
- (a2a9b34) [Merge pull request #48 from grosser/grosser/bump](https://github.com/grosser/soft_deletion/commit/a2a9b34ce96f1c8d43702024f5868aad7793e483)
- (5083bf1) [add support for rails 7](https://github.com/grosser/soft_deletion/commit/5083bf125c2947eb92d806d9abf8b7902eed5e38)
- (da1019b) [Merge pull request #47 from roman-melnyk/master](https://github.com/grosser/soft_deletion/commit/da1019b86ac94a4bc510f12726475919683f359f)
- (49d3264) [v1.7.0](https://github.com/grosser/soft_deletion/commit/49d3264614c588393230d41c614b4ce7fa75827b)
- (4be3993) [Ruby 3 support](https://github.com/grosser/soft_deletion/commit/4be3993324629ed5af2d72daa59caef9f72b9716)
- (d866834) [Merge pull request #49 from MarkusHarmsen/master](https://github.com/grosser/soft_deletion/commit/d866834d543db628bd0b377efef468ab45c67e35)
- (f1092eb) [bump version when bumping](https://github.com/grosser/soft_deletion/commit/f1092eb77d50d88af9ef07662db37d523347e731)
- (ec0f7dd) [v1.8.0](https://github.com/grosser/soft_deletion/commit/ec0f7dd06f5a546e42d955e5dc1b5427834509eb)


</details>